src/FtlServer.h

```diff
@@ -66,7 +66,7 @@ class FtlServer : public FtlControlConnectionManager
     /**
      * @brief Starts listening for FTL connections on a new thread.
      */
-    void StartAsync();
+    void StartAsync(bool nackLostPackets);
```

Comment:
Any reason `nackLostPackets` can't be a constructor parameter?

Reply:
Yeah it is weird. The issue is FtlServer is constructed before configuration is loaded in `Init`.
https://github.com/Glimesh/janus-ftl-plugin/blob/master/src/JanusFtl.cpp#L40

We could just move construction of the FtlServer into Init? I think that makes sense, though I'd also then move the control listener construction Init.